src/core/toxid.h

```diff
@@ -41,9 +41,9 @@ class ToxId
 public:
     ToxId(); ///< The default constructor. Creates an empty Tox ID.
     ToxId(const ToxId& other); ///< The copy constructor.
-    ToxId(const QString& id); ///< Create a Tox ID from QString.
-                              /// If the given id is not a valid Tox ID, then:
-                              /// publicKey == id and noSpam == "" == checkSum.
+    explicit ToxId(const QString& id); ///< Create a Tox ID from QString.
+                                       /// If the given id is not a valid Tox ID, then:
+                                       /// publicKey == id and noSpam == "" == checkSum.
```

Comment:
In real, I want (because it's faster)
Until qtox hasn't any API docs moving docs to cpp made work horrible IMHO.
If you still think, that it should be moved to cpp, I do it in next PR, Ok?